go.mod

```diff
@@ -9,12 +9,14 @@ require (
 	github.com/cznic/y v0.0.0-20170802143616-045f81c6662a
 	github.com/go-sql-driver/mysql v1.3.0
 	github.com/pingcap/check v0.0.0-20190102082844-67f458068fc8
-	github.com/pingcap/errors v0.11.5-0.20201029093017-5a7df2af2ac7
-	github.com/pingcap/log v0.0.0-20200511115504-543df19646ad
+	github.com/pingcap/errors v0.11.5-0.20210513014640-40f9a1999b3b
+	github.com/pingcap/log v0.0.0-20210625125904-98ed8e2eb1c7
+	github.com/pkg/errors v0.9.1 // indirect
 	github.com/remyoudompheng/bigfft v0.0.0-20190728182440-6a916e37a237 // indirect
-	go.uber.org/atomic v1.7.0 // indirect
-	go.uber.org/zap v1.15.0
-	golang.org/x/text v0.3.1-0.20180807135948-17ff2d5776d2
+	go.uber.org/atomic v1.8.0 // indirect
+	go.uber.org/multierr v1.7.0 // indirect
```

Comment:
can these 3 indirect dependencies be removed?

Reply:
```
$ grep '// indirect' go.mod
	github.com/cznic/golex v0.0.0-20181122101858-9c343928389c // indirect
	github.com/pkg/errors v0.9.1 // indirect
	github.com/remyoudompheng/bigfft v0.0.0-20190728182440-6a916e37a237 // indirect
	go.uber.org/atomic v1.8.0 // indirect
	go.uber.org/multierr v1.7.0 // indirect
```
All 5 of them? or only specific ones? why?

Reply:
since we are not directly using them it seems inappropriate to require a higher version than necessary.

well the `golex` and `bigfft` deps refuse to be gone after `go mod tidy`, so yeah only these three.